plugins/woocommerce/includes/wc-conditional-functions.php

```diff
@@ -281,6 +295,18 @@ function is_lost_password_page() {
 	}
 }
 
+if ( ! function_exists( 'is_admin_settings_page' ) ) {
+
+	/**
+	 * is_admin_settings_page - Returns true when viewing the admin settings page.
+	 *
+	 * @return bool
+	 */
+	function is_admin_settings_page() {
+		return is_admin() && isset( $_GET['page'] ) && 'wc-settings' === $_GET['page']; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	}
```

Comment:
This function name feels too generic -- as above, I think we need to include `wc` in the function name somewhere, especially given that just reading the function name and comment could easily make someone assume that the code is checking for `$_GET['page'] === 'settings'`.

Reply:
Fair point! I have renamed it to `is_wc_admin_settings_page` following the same convention we have for `is_wc_endpoint`